thejamesgore/Foxtons-Dashboard
# feat: Add dark mode to site and sidebar

## Description 🚀

This pull request adds a new dark mode option to the site and sidebar components. The dark mode is toggled using a button in the top-right corner of the page, which switches the background color of the site and sidebar to a darker color scheme.

By adding a dark mode option, we provide a more customizable user experience and enable users to better tailor the application to their preferences.

## Changes Made 👉

- Added a new dark mode option to the site and sidebar components.
- Created a button that toggles the background color of the site and sidebar to a darker color scheme.

## Reason for Changes 🔍

The addition of a dark mode option improves the user experience and provides a more customizable interface for users.

## How to Test

To test these changes, you can follow these steps:

1. Clone this branch to your local machine.
2. Install the necessary dependencies with `npm install`.
3. Start the application with `npm start`.
4. Verify that the application starts without any errors.
5. Toggle the dark mode option using the button in the top-right corner of the page.
6. Verify that the background color of the site and sidebar changes to a darker color scheme.

## Checklist


## Related Issues

None.

## Other Notes

None.
